### DISCUSSION TITLE
[Esame 22 gennaio 2024] Esercizio 1

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#442] [Esami M-Z]
![22 gennaio es. 1](https://github.com/user-attachments/assets/2f50a86a-da4a-4d74-9369-293fdcaf9602)

<details>
  <summary>soluzione del prof</summary>
  <br>
  <img src="https://github.com/user-attachments/assets/9119a76d-2001-423a-ad01-284f1c463479" alt="soluzione"  />
  <img src="https://github.com/user-attachments/assets/694c1548-e69a-4fe0-bf2d-7f3fa3c69019" alt="soluzione 2"  />

</details>

**Comment by thegeek-sys** [comment]
![immagine](https://github.com/user-attachments/assets/7f36dde5-82d7-47b9-bce1-2ae4e79f93fe)

**Comment by niccolo-fato** [comment]
<img width="1280" height="1058" alt="image" src="https://github.com/user-attachments/assets/7f499705-d401-43ad-9a9d-3011843f2158" />
